docs/assets/js/main.js

```diff
@@ -0,0 +1,14 @@
+let acc = document.getElementsByClassName("video_accordion");
```

Comment:
Endeavor to use const whenever you're not reassigning a variable. It makes the intention clearer.

acc is not clear enough. It could be renamed to videoAccordions